app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,6 @@
+class FoodsController < ApplicationController
+  def index
+    @foods = FoodFacade.find_foods_by_ingredient(params[:q])
+    @number_of_results = FoodFacade.find_number_of_results(params[:q])
+  end
```

Comment:
Both of these methods, `find_foods_by_ingredient` and `find_number_of_results` call the same endpoint, and just pull different data. I'd like to challenge you to find a way to just call one method in your facade, and have your facade take care of combining the food objects and the totalHits into one object to get returned to your controller. Ideally, you just have one object getting sent to your view that contains all data necessary for that view. And in this case, all of the data that you need for this view, can come from the one query call you're making.

Reply:
You could take a look at this syntax: 

`@number_of_results, @foods = FoodFacade.your_method(params[:q])`

Where `your_method` returns a 2d array, with the number of results as the first element and the food collection as the second element.